### PR TITLE
ttl dataset file

### LINKED ISSUE #792: Display Text That Explains TTL For Uploaded Dataset Files
**Describe the solution you'd like**
Display a text in the dataset step under the upload button that explains that the TTL (Time To Live) for uploaded datasets files is 5 days ("files will be deleted after 5 days" or something like that).

**Additional context**
[https://mui.com/material-ui/react-typography/](https://mui.com/material-ui/react-typography/)

**Setup Instructions (what branch to work off of)**
Run the following commands
```
git checkout nextjs
git pull origin nextjs
git checkout -b dataset-file-ttl-text
```
FYI: If you are not able to immediately run `git checkout nextjs`, make sure you commit your changes in the current branch or run `git stash` and then execute the above commands

### PR DESCRIPTION
**Pull request by codingwithsurya**
# ttl dataset file

uploaded file will be deleted after 5 days

provides that messaging

verify text shows up on the page

### CODE REVIEW

File: frontend/src/features/Train/components/DatasetStepLayout.tsx
Comment (farisdurrani) [inline frontend/src/features/Train/components/DatasetStepLayout.tsx]: Minor comment: does it really need to be a heading over here? Headings are supposed to be for titles and headers, not info text. 

Also try not to use negative margins. Reduce the margin/padding on the preceding component instead

**Review by karkir0003 — COMMENTED** [review]
conditionally approve @yeaitsurya . wait for review from @dwu359 and @farisdurrani

**Review by farisdurrani — APPROVED** [review]

**Pull request merged**
